README.md

```diff
@@ -452,7 +453,7 @@ British Machine Vision Conf. (BMVC), 2020. [PDF](https://www.bmvc2020-conference
 - <a name="Liu21arxiv"></a>Liu, Z., Fu, Y.,  
 *[e-ACJ: Accurate Junction Extraction For Event Cameras](https://arxiv.org/pdf/2101.11251)*,  
 arXiv, 2021.
-- <a name="Dong21ICMVA"></a>Dong, Y., Zhang, T.,  
+- <a name="Dong21ICMVA"></a>Dong, Y. and Zhang, T.,  
```

Comment:
we use lowercase for publication venue: Dong21icmva

Reply:
Done. This is an original issue, existed before my first pull request.

Reply:
thanks